packages/graphql/src/dataloaders/Locationsloader.js

```diff
@@ -0,0 +1,45 @@
+// @flow
+
+import Dataloader from 'dataloader';
+import stringify from 'json-stable-stringify';
+import qs from 'querystring';
+
+import fetch from '../services/Fetch';
+
+type ApiResponse = {|
+  +locations: $ReadOnlyArray<{|
+    +id: string,
+    +name: string,
+    +slug: string,
+  |}>,
+|};
+
+export type Location = {|
+  +id: string,
+  +name: string,
+  +slug: string,
+|};
+
+export type Locations = $ReadOnlyArray<Location>;
+
+function sanitizeLocations(locations: $PropertyType<ApiResponse, 'locations'>) {
+  return locations.map(location => ({
+    id: location.id,
+    name: location.name,
+    slug: location.slug,
+  }));
+}
+
+const fetchLocations = async (ids: $ReadOnlyArray<{| +term: string |}>) => {
+  const data = await Promise.all(
+    ids.map(async ({ term }) =>
```

Comment:
the async keyword should no longer be necessary here.

Reply:
👍

Reply:
done